docs/governance/README.md

```diff
@@ -0,0 +1,127 @@
+    considered rejected.
+  - A **stable implementation** in at least one .NET implementation. The
+    implementation must be licensed under an open source license that is
+    compatible with MIT. This will allow other .NET implementations to jump-
+    start their own implementations or simply take the feature as-is.
+* **.NET Standard updates are planned** and will generally follow a set of
+  themes. We avoid grab-bag releases of .NET Standard and instead try to define
+  a set of goals that describe what kind of feature areas a particular .NET
+  Standard version provides. This simplifies answering the question which .NET
+  Standard a given library should depend on. It also makes it easier for .NET
+  implementations to decide whether it's worth implementing a higher version of
+  .NET Standard.
+* **The version number** is subject to discussion and is generally a function of
+  how significant the new version is. While we aren't planning on making
+  breaking changes, we'll rev the major version if the new version adds large
+  chunks of APIs or has sizable changes in the overall developer experience.
+
+[netstandard-api]: https://github.com/dotnet/standard/issues?q=is%3Aopen+is%3Aissue+label%3Anetstandard-api
+
+## Inclusion principles
+
+Not every .NET API needs to be (or even should be) part of the .NET Standard.
+
+If a library is written in pure IL (for example C#, VB.NET, F# etc.), and
+targets .NET Standard, then it can run across all current and future .NET
+implementations with no code changes. The only requirement on the .NET
+implementation is that it has to support at least the version of .NET Standard
+that the library was compiled for (or a higher version).
+
+Any modifications to such a library, whether they're bug fixes, performance
+improvements, or additional APIs, will also work across all these .NET
+implementations as well. That means the library author has flexibility to make
+releases on their own cadence while consumers of the library have the choice on
+when they upgrade the version they are using. If the library is made a part of
+.NET Standard, it can only version at the same cadence as the standard, so the
+update flexibility is much reduced.
+
+On the other hand, there are also downsides for APIs not being part of the
+standard. If an API ships as a library, it's much harder for any .NET
+implementation to use the types as part of their implementation or public API
+surface. Hence, there is a lot of value in adding widely used concepts to the
+.NET Standard in order to use them throughout all .NET implementations.
+
+The following criteria helps us to identify APIs that should be part of .NET
+Standard:
+
+* **Ubiquitous APIs**. APIs in the .NET Standard must be implemented by all .NET
+  implementations. Thus, we're only interested in standardizing APIs that are
+  universal in nature and thus should be available everywhere. Of course, this
```

Comment:
By "everywhere" here, I assume this means "all .NET implementations" rather than "all OSes".  There are / will be APIs that are in all .NET implementations but that don't run everywhere.

Reply:
That's fair, I'll clarify.